app/src/main/java/ar/com/wolox/android/cookbook/navigation/NavigationExtension.kt

```diff
@@ -0,0 +1,24 @@
+package ar.com.wolox.android.cookbook.navigation
+
+import androidx.navigation.NavOptions
+import ar.com.wolox.android.cookbook.R
+
+class NavigationExtension {
+
```

Comment:
We should add an extension method for `Fragment` which is called `requireNavController` so to avoid the repeated `Navigation.findNavController(view!!)`. Something like this:

```
fun Fragment.requireNavControler() = Navigation.findNavController(view!!)!!
```

This way we force the non nullability of the `NavController`.